src/precice/tests/explicit-bounding-box.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<precice-configuration>
+ <solver-interface dimensions="2">
+    <data:scalar name="Velocities" />
+
+    <mesh name="Test-Square">
+      <use-data name="Velocities" />
+    </mesh>
+
+    <mesh name="MeshOne">
+      <use-data name="Velocities" />
+    </mesh>
+
+    <participant name="SolverOne">
+      <use-mesh name="MeshOne" provide="yes" />
+      <use-mesh name="Test-Square" from="SolverTwo" />
+      <write-data name="Velocities" mesh="MeshOne" />
+      <mapping:solver
+        direction="write"
+        from="MeshOne"
+        to="Test-Square"
+	 constraint="consistent" />
+	 <!--TODO: remove constraint-->
+    </participant>
```

Comment:
I guess easier would be: (this is also how it was working in the server-based preCICE)
```suggestion
    <participant name="SolverOne">
      <use-mesh name="Test-Square" from="SolverTwo" />
      <write-data name="Velocities" mesh="Test-Square" />
    </participant>
```